docs/meshstack.aws.aad-as-idp.md

```diff
@@ -0,0 +1,80 @@
+---
+id: meshstack.aws.aad-as-idp
+title: AAD as Identity Provider
+---
+## Configure AAD as IdP for AWS SSO
+
+You can trigger everything that is needed for using AAD as an IdP for AWS SSO from within Azure.
+
+### Create an Enterprise Registration
+
+Search for `aws` and choose `AWS Single Sign-on` as the enterprise registration type
+
+![assets/aad_as_idp_for_aws/aad_as_idp_for_aws_01.png](assets/aad_as_idp_for_aws/aad_as_idp_for_aws_01.png)
+
+Follow the steps in AAD under Getting Started (Pick SAML for the single sign on.)
+
+![assets/aad_as_idp_for_aws/aad_as_idp_for_aws_02.png](assets/aad_as_idp_for_aws/aad_as_idp_for_aws_02.png)
+
+### Configure AWS for AAD integration
+
+Choose `AWS Single Sign-on`
```

Comment:
d: The switch to AWS Console should also be explicitly defined in my opinion - it also conflicts with the first sentence that `You can trigger everything that is needed for using AAD as an IdP for AWS SSO from within Azure.`

Reply:
yeah, i also think that the sentence at the beginning that says everything is done in AAD does not fit to this guide here. You don't use the browser plugin, but you go the way of configuring everything manually, which is fine. We just need a different introduction then

Reply:
d: I think you have to mention here that AWS SSO must be enabled initially for the root account. I am not sure how that looks like, you may need to check AWS docs for that, but i think a special screen where you have to enable AWS SSO pops up somehow. Not sure whether something needs to be filled out there, or whether you can just say "enable it"

Reply:
I now explicitly point out that this must be done in the AWS console. The step that SSO has to be enabled is now also available.